src/os/ObjectStore.h

```diff
@@ -2014,7 +2014,7 @@ class ObjectStore {
    * @param aset map of keys and buffers that contain the values
    * @returns 0 on success, negative error code on failure
    */
-  virtual int collection_getattrs(coll_t cid, map<string,bufferptr> &aset)
+  int collection_getattrs(coll_t cid, map<string,bufferptr> &aset)
```

Comment:
I'm pretty sure we don't want to eliminate these deprecated warnings like this. Removing the "virtual" is disassociating them from the parent class' function table, which is definitely not right. We just really did deprecate them.

Granted, I'm not a big fan of making the build throw warnings by keeping stuff like this around; @liewegas what's the plan here?